3/debian-10/rootfs/opt/bitnami/scripts/libmoodle.sh

```diff
@@ -350,3 +353,25 @@ if (isset(\$_SERVER['HTTPS']) \&\& \$_SERVER['HTTPS'] == 'on') {\\
 }"
     replace_in_file "$MOODLE_CONF_FILE" "\\\$CFG->wwwroot\s*=.*" "$conf_to_replace"
 }
+
+moodle_configure_reverseproxy() {
+
+    # Checking the reverseproxy setting values
+    if ! is_empty_value "$MOODLE_REVERSEPROXY"; then
+        if [[ "$MOODLE_REVERSEPROXY" == "true" ]]; then
+            echo "\$CFG->reverseproxy = true;" >> "$MOODLE_CONF_FILE"
+        elif [[ "$MOODLE_REVERSEPROXY" != "false" ]]; then
+            warn "The allowed values for MOODLE_REVERSEPROXY are: true or false"
+        fi
+    fi
+
+    # Checking the sslproxy setting values
+    if ! is_empty_value "$MOODLE_SSLPROXY"; then
+        if [[ "$MOODLE_SSLPROXY" == "true" ]]; then
+            echo "\$CFG->sslproxy = true;" >> "$MOODLE_CONF_FILE"
+        elif [[ "$MOODLE_SSLPROXY" != "false" ]]; then
+            warn "The allowed values for MOODLE_SSLPROXY are: true or false"
+        fi
+    fi
```

Comment:
In the **README.md**, you mentioned that the default value is "no". On the other hand, the env. var validation should be done in `moodle_validate` function: 
```suggestion
    # Checking the reverseproxy setting values
    is_boolean_yes "$MOODLE_REVERSEPROXY" && echo "\$CFG->reverseproxy = true;" >> "$MOODLE_CONF_FILE"
    # Checking the sslproxy setting values
    is_boolean_yes "$MOODLE_SSLPROXY" && echo "\$CFG->sslproxy = true;" >> "$MOODLE_CONF_FILE"
    # Avoid exit code of previous commands to affect the result of this function
    true
```

And add to `moodle_validate`:
```bash
    check_yes_no_value() {
        if ! is_yes_no_value "${!1}" && ! is_true_false_value "${!1}"; then
            print_validation_error "The allowed values for ${1} are: yes no"
        fi
    }
    ...
    check_yes_no_value "MOODLE_REVERSEPROXY"
    check_yes_no_value "MOODLE_SSLPROXY"
```